tests/spec/core/biblio-db-spec.js

```diff
@@ -192,7 +200,9 @@ describe("Core - biblioDB", () => {
       var p2 = biblioDB.isAlias(null);
       var p3 = biblioDB.isAlias("");
       await Promise.all([
-        p1.catch(err => expect(err instanceof TypeError).toBe(true)),
+        p1.catch(err =>
+
+        expect(err instanceof TypeError).toBe(true)),
```

Comment:
This looks strange, what eslint rule forces this?

Reply:
new-line-before-expect | 1 |

Reply:
yeah, that's the "buggy" one... the test could be rewritten to use async/await tho, in which case it would look right. Or could be refactored to use `expect().throws` (can't remember the exact Jasmine method).